wire-schema/src/test/java/com/squareup/wire/schema/internal/parser/EnumElementTest.kt

```diff
@@ -0,0 +1,186 @@
+    val element = EnumElement.builder(location)
+        .name("Enum")
+        .options(ImmutableList.of(OptionElement.create("kit", STRING, "kat")))
+        .constants(
+            ImmutableList.of(
+                EnumConstantElement.builder(location).name("ONE").tag(1).build(),
+                EnumConstantElement.builder(location).name("TWO").tag(2).build(),
+                EnumConstantElement.builder(location).name("SIX").tag(6).build()
+            )
+        )
+        .build()
+    val expected = """
+        |enum Enum {
+        |  option kit = "kat";
+        |
+        |  ONE = 1;
+        |  TWO = 2;
+        |  SIX = 6;
+        |}
+        |""".trimMargin()
+    assertThat(element.toSchema()).isEqualTo(expected)
+  }
+
+  @Test
+  fun addMultipleOptions() {
+    val kitKat = OptionElement.create("kit", STRING, "kat")
+    val fooBar = OptionElement.create("foo", STRING, "bar")
+    val element = EnumElement.builder(location)
+        .name("Enum")
+        .options(ImmutableList.of(kitKat, fooBar))
+        .constants(
+            ImmutableList.of(
+                EnumConstantElement.builder(location).name("ONE").tag(1).build()
+            )
+        )
+        .build()
+    assertThat(element.options()).hasSize(2)
+  }
+
+  @Test
+  fun simpleWithDocumentationToSchema() {
+    val element = EnumElement.builder(location)
+        .name("Enum")
+        .documentation("Hello")
+        .constants(
+            ImmutableList.of(
+                EnumConstantElement.builder(location).name("ONE").tag(1).build(),
+                EnumConstantElement.builder(location).name("TWO").tag(2).build(),
+                EnumConstantElement.builder(location).name("SIX").tag(6).build()
+            )
+        )
+        .build()
+    val expected = """
+        |// Hello
+        |enum Enum {
+        |  ONE = 1;
+        |  TWO = 2;
+        |  SIX = 6;
+        |}
+        |""".trimMargin()
+    assertThat(element.toSchema()).isEqualTo(expected)
+  }
+
+  @Test
+  fun fieldToSchema() {
+    val value = EnumConstantElement.builder(location)
+        .name("NAME")
+        .tag(1)
+        .build()
+    val expected = "NAME = 1;\n"
+    assertThat(value.toSchema()).isEqualTo(expected)
+  }
+
+  @Test
+  fun fieldWithDocumentationToSchema() {
+    val value = EnumConstantElement.builder(location)
+        .name("NAME")
+        .tag(1)
+        .documentation("Hello")
+        .build()
+    val expected =
+      """// Hello
+        |NAME = 1;
+        |""".trimMargin()
+    assertThat(value.toSchema()).isEqualTo(expected)
+  }
+
+  @Test
+  fun fieldWithOptionsToSchema() {
+    val value = EnumConstantElement.builder(location)
+        .name("NAME")
+        .tag(1)
+        .options(
+            ImmutableList.of(
+                OptionElement.create("kit", STRING, "kat", true),
+                OptionElement.create("tit", STRING, "tat")
+            )
+        )
+        .build()
+    val expected = """NAME = 1 [
```

Comment:
Does the test fail when `NAME = 1 [` is on the next line? If not, it shouldn't be on the same line as `"""`.